ietf-optical-impairment-topology.yang

```diff
@@ -605,42 +600,252 @@ module ietf-optical-impairment-topology {
           fraction-digits 2;
         }
         units dB;
-    description "connector-out"; 
+    description "connector-out";
       }
     }
   }
 
-  grouping roadm-params{
-    description "roadm parameters description";
-    container roadm{
-    description "roadm parameters";
-      leaf type_variety {
-        type string ;
-        mandatory true ;
+ 
+
+  grouping roadm-express-path {
+    description "roadm express path optical impairments";
+
+    container roadm-express-path {
+      description "roadm parameters per express path";
+     
+      leaf roadm-pmd {
+        type decimal64 {
+          fraction-digits 8;
+          range "0..max";
+        }
+        units "ps/(km)^0.5"; 
+        description 
+          "Polarization Mode Dispersion";
+      }
+      leaf roadm-cd {
+        type decimal64 {
+          fraction-digits 5;
+        }
+        units "ps/nm/km";
+        description "Cromatic Dispersion";
+      }            
+      leaf pdl {

```

Comment:
If would suggest to use the same wording as roadm-xxx -> roadm-pdl

Reply:
done